docs/configuration/configuration.md

```diff
@@ -15,7 +15,7 @@ Supported flags are as follows (result of running `./flagd start --help`):
   -k, --server-key-path string    Server side tls key path
   -a, --sync-provider-args        Sync provider arguments as key values separated by =
```

Comment:
Are sync provider args still used?

Reply:
They are not, however I was hesitant to remove them completely, viper will throw if an unexpected argument is passed, I can open an issue to follow the pattern of `sync-provider` where a warning is thrown? However we may be 'reintroducing' args in future development

Reply:
We can address this issue [here](https://github.com/open-feature/flagd/issues/233).